src/backend/src/lib.rs

```diff
@@ -398,6 +400,11 @@ fn get_user_profile() -> Result<UserProfile, GetUserProfileError> {
     })
 }
 
+#[query(guard = "may_read_user_data")]
+async fn allow_signing() -> Result<(), AllowSigningError> {
```

Comment:
Will this be used only for signing?

We also use the CFS to get the address and balance for example.

Reply:
Done.